nds-org/workbench-webui
# fix: downgrade react-scripts to 4.0.3 (5.0.1 had issues)

## Problem
Temporary fix for #18 

SwaggerUI page is broken, missing the `Buffer` polyfill

## Approach
* Downgrade `react-scripts` to `4.0.3` to bring back this polyfill
* Note: webpack is not customizable without ejecting webpack config

## How to Test
1. Checkout and run this branch locally
2. Navigate to http://localhost:3000/swagger
    * Page should load
    * You should no longer see the `Buffer is not defined` error in the console
